### ISSUE TITLE
Does not seem to work at all?

### ISSUE THREAD

**Issue by hoegge** [pR0Ps/grafana-trackmap-panel#70]
Just installed the plugin from the Grafana site:

https://grafana.com/grafana/plugins/pr0ps-trackmap-panel/

![image](https://user-images.githubusercontent.com/15846310/136981648-2eaf9517-a631-49b4-8ce3-220dfc01b53f.png)

and restarted the Grafana server (runs in docker)

The map shows up in Grafana but the panel is just black and there are no options.

What can I try to find the problem?

**Comment by iandees** [comment]
I'm also having this problem. I'm running Grafana 8.2.2. The query is returning data with labels "latitude" and "longitude" successfully. What other information would be helpful to diagnose?

**Comment by hakkimakki** [comment]
Haveing the same problem. I'm running Grafana 8.3.3 with Docker. 

<img width="1280" alt="error" src="https://user-images.githubusercontent.com/38659001/148040713-5e708847-8bb2-4383-9c15-d5cbc392c0f9.PNG">

However the Query returns the following results:

<img width="1279" alt="error_2" src="https://user-images.githubusercontent.com/38659001/148040771-6606bcc6-cd98-4a2c-bbd8-48a8adf2c4fd.PNG">

The following message apeared in Google Chrome which may help to find the problem:

![image](https://user-images.githubusercontent.com/38659001/148040908-fe1d5672-8c41-4e10-90c4-04818a3334f2.png)

**Comment by CourchesneA** [comment]
Any progress here ? Did someone find a fix other than duplicating the query ?

**Comment by dmak** [comment]
Same issue after upgrading Grafana from 7.x to 8.x – the plugin stopped working. It shows the map / globe, but does not show the track. Query returns the valid data as the query works flawlessly with built-in Geomap visualization.

**Comment by bauen1** [comment]
I think I've found an issue with

https://github.com/pR0Ps/grafana-trackmap-panel/blob/974b0d29078696c2990c96f502e6bddca78c80df/src/trackmap_ctrl.js#L375-L380

`      this.leafMap.setView([0, 0], 1);` is never called if data is present, this leads to no map appearing, even if you have correct queries.

I've tested this by debugging in firefox, adding a breakpoint on the `onDataReceived` method, when it is hit, switch to console, do
`a = this` so it can be accessed later. Then hit continue. After this run `a.leafMap.setView([0, 0], 1);` in the console and watch your data appear.
